pkg/shell/cockpit-docker.js

```diff
@@ -1839,9 +1869,32 @@ PageContainerDetails.prototype = {
 
     start_container: function () {
         var self = this;
+        var id = this.container_id;
         this.client.start(this.container_id).
                 fail(function(ex) {
-                    shell.show_unexpected_error(ex);
+                    var end_phrase = '.scope already exists';
```

Comment:
Why do we copy and paste so much code all over again? It'd be fine to have file scope (global, but not really window level global) functions, so we can avoid code duplication. Or is there a reason for copy/paste I'm missing?